sdk/resourcemanager/azure-resourcemanager-compute/CHANGELOG.md

```diff
@@ -2,8 +2,9 @@
 
 ## 2.2.0-beta.1 (Unreleased)
 
-- Updated `api-version` to `2020-06-01`
-- Supported virtual machines force delete
+- Updated `api-version` to `2020-12-01`
+- Supported force deletion on virtual machines and virtual machine scale sets
+- Removed container service as it is deprecated in compute, please use `KubernetesCluster` from `azure-resourcemanager-containerservice`
```

Comment:
@ChenTanyi Let me know if this line is correct.

Reply:
I think it is OK.